### PR TITLE
docs: add note for auth code flow limitations on mobile

### PR DESCRIPTION
**Pull request by SLdragon**
https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/17403430

### CODE REVIEW

**Review by KennethBWSong — DISMISSED** [review]

File: templates/scenarios/js/sso-tab/README.md [hunk 1]
```diff
@@ -84,3 +84,8 @@ Once deployed, you may want to distribute your application to your organization'
 
 - From Visual Studio Code: open the Teams Toolkit and click `Publish to Teams` or open the command palette and select: `Teams: Publish to Teams`.
 - From TeamsFx CLI: run command `teamsfx publish` in your project directory.
+
+## Current limitations on mobile platform
+- On IOS (and a few Android) mobile platform, if user close the Teams App, he would have to sign in again due to browser [blocks third-party cookies](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more) and this results in additional prompts. Related issue can be found [here](https://github.com/OfficeDev/TeamsFx/issues/7574).
+
```
Comment (MuyangAmigo) [inline templates/scenarios/js/sso-tab/README.md:90]: IOS -> iOS

Comment (MuyangAmigo) [reply templates/scenarios/js/sso-tab/README.md:90]: Or can we say something like: 

This template has adopted [Authorization Code Flow](https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow) to implement SSO. There's a known limitation with mobile platforms such as iOS and a few Android versions where users have to sign in again as the state was not stored due to [third party cookie being blocked by the browser](https://webkit.org/blog/10218/full-third-party-cookie-blocking-and-more).

To achieve full SSO experience, we will recommend you to implement OBO flow to pass your token to a server. See more [here](https://learn.microsoft.com/en-us/microsoftteams/platform/tabs/how-to/authentication/tab-sso-code#pass-the-access-token-to-server-side-code)

Comment (MuyangAmigo) [reply templates/scenarios/js/sso-tab/README.md:90]: Also I think we can add this as a note after prerequisite section.

**Review by MuyangAmigo — CHANGES_REQUESTED** [review]

File: templates/scenarios/js/sso-tab/README.md (hunk 1 above)
Comment (SLdragon) [reply templates/scenarios/js/sso-tab/README.md:90]: Sure

Comment (SLdragon) [reply templates/scenarios/js/sso-tab/README.md]: Updated

**Review by KennethBWSong — APPROVED** [review]

**Review by MuyangAmigo — APPROVED** [review]

**Review by hund030 — APPROVED** [review]

**Pull request merged**